### ISSUE TITLE
Rethink definition / term relationship and naming

### ISSUE THREAD

**Issue by scottaohara** [w3c/aria#2074]
The relationship for a term/definition is defined in the spec by creating an aria-labelledby or nesting relationship with the definition to the term.  

If I recall correctly, the naming idea was so that a definition could be associated with its term.  but looking at this now as i'm going through the roles / how they can be named, i'm not really sure why we didn't use aria-details instead?

Also, I thought it odd that at term could get name from author, but i realize that might be because of these previously being considered standins for `dt` and `dd` elements, but that was also rethought when working on aria 1.2.

**Comment by scottaohara** [comment]
looking at the simplified explainer to move aria annotations into the aria spec proper, it already states that
>A term should be linked to its defintion using aria-details rather than aria-labelledby.

it seems that part of my issue is already answered, in that this part of the spec should change.

now the question remains if term/definition should allow for aria-label/labelledby at all.  i lean towards 'no'.

**Comment by scottaohara** [comment]
updated spec text should talk about how a details relationship between these elements is necessary only in situations where the association might not be clear.  otherwise, it is not a requirement for authors to make the association.

otherwise, these roles should be changed to be name prohibited.

**Comment by Arsikod** [comment]
Is there any decision recommended way to keep relation between dt and dd?
I have a practical question because we are using key-value pairs everywhere.

I have a component that shows name-value pairs. Previously I was using `aria-labelledby`, and could test my DefinitionItem cohesively so I know which dt is the key of dd value

🧪here is the test for below component

`expect(screen.getByLabelText(/key/i)).toHaveTextContent(/value/i);`

```
function DefinitionItem({ term, description }) {
  const termId = useId();

  return (
    <>
      <dt id={termId}>{term}</dt>
      <dd aria-labelledby={termId}>{description}</dd>
    </>
  );
}
```

But since it is prohibited in [wai aria 1.3 draft](https://w3c.github.io/aria/#definition) I decided to go with `aria-describedby` since it is the only way to keep relationship between these items and being able to query by accessible role.
so here is how my test looks now:

```
const ddElement = screen.getByRole("definition", { //🍒i still can benefit from role
  description: /text in dt term tag/i,
});

expect(ddElement).toHaveTextContent(/text in dd definition tag/i);
```
and component uses `aria-describedby`

```
function DefinitionItem({ term, description }) {
  const termId = useId();

  return (
    <>
      <dt id={termId}>{term}</dt>
      <dd aria-describedby={termId}>{description}</dd> //🍒adds link to dt term
    </>
  );
}
```

do you think this is proper approach at least for now, while there is a still no certain decision how to keep name-value related?
